plugins/discovery-multicast/src/main/java/org/elasticsearch/plugin/discovery/multicast/MulticastDiscoveryPlugin.java

```diff
@@ -44,8 +46,9 @@ public String name() {
     public String description() {
         return "Multicast Discovery Plugin";
     }
-    
+
     public void onModule(DiscoveryModule module) {
+        deprecationLogger.deprecated("[discovery-multicast] plugin will be removed in 5.0.0. Use unicast or any cloud discovery plugin.");
```

Comment:
I wondered the same. But we know for sure that this one has been removed in 5.0.0. We are may be unsure for the other ones (like mapper-attachments which will most likely be removed in 6.0.0 or 5.1.0).
But ok, I'll fix it and make it "standard".